channels/auth.py

```diff
@@ -17,10 +20,11 @@
 from channels.db import database_sync_to_async
 from channels.middleware import BaseMiddleware
 from channels.sessions import CookieMiddleware, SessionMiddleware
+from channels.utils import StrDict
 
 
 @database_sync_to_async
-def get_user(scope):
+def get_user(scope: StrDict) -> Awaitable[Union[AbstractBaseUser, AnonymousUser]]:
```

Comment:
`get_user()` doesn't return an awaitable.

Reply:
I'm sorry, I don't understand. In every reference in the codebase there's await before and database_sync_to_async decorator makes the return value of the function an awaitable, doesn't it?

Reply:
see my comment above, the `@database_sync_to_async` is making it `Awaitable` so the return signature of the `def get_user ` is not  `Awaitable`

Reply:
Okay, I'll fix it